geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RemoteTransactionDUnitTest.java

```diff
@@ -3695,42 +3596,26 @@ public void afterInvalidate(EntryEvent event) {
     private boolean oneCreate;
 
     public void checkSuccess() {
-      if (oneDestroy && oneCreate) {
-        // chill
-      } else {
-        fail("Didn't get both events. oneDestroy=" + oneDestroy + " oneCreate=" + oneCreate);
-      }
+      assertThat(oneDestroy && oneCreate).isTrue();
```

Comment:
The failure message is lost with this change. It would be good to keep it using `as()`.